### PR TITLE
Feature/new coat of paint

### PR DESCRIPTION
**Pull request by gfodor**
This PR restyles the whole 2d part of the app based on the new design

### CODE REVIEW

File: src/hub.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <link rel="shortcut icon" type="image/png" href="/favicon.ico">
     <title>Get together | Hubs by Mozilla</title>
     <link href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,400i,700" rel="stylesheet">
```
Comment (brianpeiris) [inline src/hub.html]: It looks like we've stopped using Zilla Slab completely. Might as well remove this font reference and cut the dead weight?

File: src/react-components/help-dialog.js [hunk 2]
```diff
@@ -12,23 +12,14 @@ export default class HelpDialog extends Component {
...
+            In VR, <b>look up</b> to find your menu.
           </p>
           <p>
-            The <b>Pause/Resume Toggle</b> pauses all other avatars. You can then block them from having further
-            interactions with you.
+            The <b>Mic Toggle</b> mutes your mic.
           </p>
           <p>
-            The <b>Bubble Toggle</b> hides avatars that enter your personal space.
+            The <b>Pause/Resume Toggle</b> pauses all other avatars and lets you block others or remove objects.
```
Comment (brianpeiris) [inline src/react-components/help-dialog.js]: Should we change the media "Delete" buttons to read "Remove" instead? There are three of them in the HTML: https://github.com/mozilla/hubs/blob/feature/new-coat-of-paint/src/hub.html#L171

Comment (gfodor) [reply src/react-components/help-dialog.js]: ah yes, i meant to do this! thanks

File: src/react-components/home-root.js [hunk 3]
```diff
@@ -270,12 +271,8 @@ class HomeRoot extends Component {
                   >
                     <FormattedMessage id="home.privacy_notice" />
                   </a>
-                </div>
-                <div className={styles.bottom}>
-                  <div>
-                    <FormattedMessage id="home.made_with_love" />
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/react-components/home-root.js]: i know i know :/

File: src/assets/stylesheets/audio.scss [hunk 4]
```diff
@@ -43,14 +43,16 @@
       @extend %glass-text;
 
       appearance: none;
-      background-color: rgba(64, 64, 64, 0.2);
+      background-color: white;
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/assets/stylesheets/audio.scss]: these were images I forgot to convert to black, done

**Pull request merged**